hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/StreamWriteOperatorCoordinator.java

```diff
@@ -291,19 +291,19 @@ public void handleEventFromOperator(int i, OperatorEvent operatorEvent) {
   }
 
   @Override
-  public void subtaskFailed(int i, @Nullable Throwable throwable) {
-    // reset the event
-    this.eventBuffer[i] = null;
-    LOG.warn("Reset the event for task [" + i + "]", throwable);
+  public void subtaskReset(int i, long l) {
```

Comment:
Is the change Forward-compatible ?

Reply:
I believe this method `subtaskReset` was just moved as opposed to actually changed ( atleast when im examining this diff).